src/openvtb/conf/base.py

```diff
@@ -79,3 +80,8 @@
         "Full path to the GDAL library used by GeoDjango. In most circumstances, this can be left empty."
     ),
 )
+
+# URN settings
+URN_NAMESPACE = config(
+    "URN_NAMESPACE", None, help_text=("Namespace used in URNs schemas.")
```

Comment:
Shouldn't this setting be required and not have a default? Otherwise the URN fields won't function properly, right?